README.md

```diff
@@ -1,5 +1,32 @@
 ![GW Open Data Workshop](share/ODWlogo.jpeg)
 
-# GW Open Data Workshop
+# Gravitational Wave Open Data Workshop
 
-This repo is under construction
+Welcome to the Gravitational Wave Open Data Workshop.
+Here you will find the support material for the [Gravitational Wave Open Data Workshop](https://learn.gwosc.org) (ODW)
+organized by the [Gravitional Wave Open Science Center](https://gwosc.org) (GWOSC).
+This material consists on several interactive Jupyter notebooks in the [Tutorials folder](./Tutorials/), divided into topics, grouped into folders for each day of the workshop.
+There are also some quiz questions that you are asked to complete at the end of each topic.
+
+Before you dive into the notebooks, we recommend first taking a look at the [setup guide](./setup.md),
+where you will find information to configure the workspace where you will be executing the tutorials.
+
+For questions concerning the software setup, the tutorials, the workshop in general, or even for GW science questions, join our community at the [ask.igwn.org](ask.igwn.org) forum.
+Before posting new questions, check first if a similar question was already asked in the [_Open Data Workshop_ category](https://ask.igwn.org/c/open-data-workshop/5).
+
+Lastly, test yourself with the GW Data Challenge!
```

Comment:
I would move this paragraph right after the first one as it is the description of the ODW.